docs/locations.md

```diff
@@ -1,44 +1,103 @@
 
-- Github release: [https://github.com/ethereum/remix-project/releases](https://github.com/ethereum/remix-project/releases) . 
+- Github release: [https://github.com/ethereum/remix-project/releases](https://github.com/ethereum/remix-project/releases). 
 
 
 ## Embedding & Linking to Remix
 
 Remix-IDE's urls have parameters -so it is possible to specify:
-* the list of plugins you want activated
-* the theme (Dark or Light) 
-* the panels that should be minimized
-* if you want the Solidity compiler to have **optimize** enabled
+* A **list of plugins to be activated** - as well as which plugin you want to be loaded in the side panel (so it gains the "focus").
+* A **Command to be sent to a plugin** - once the plugin loads.
+* **The theme** (Dark or Light). 
+* The panels that should be **minimized**.
+* The **version of the Solidity** compiler & the **optimize** option enabled or disabled.
+* Which panels should be minimized.
+
+### Activating a list of plugins
+The following example contains the url parameter **activate** followed by **a list of plugins**. The last plugin will gain the focus.  
+
+When you use the activate list, all other plugins that a user had loaded will be deactivated.  This does not apply to the file explorer, the plugin manager, and the settings modules because these are never deactivated.
 
-In the following example, there is a list of plugins that follows the word **plugins** will be activated and the last plugin will gain the focus.
 ```
 https://remix.ethereum.org/?#activate=solidity,solidityUnitTesting,udapp,defiexplorer
 ```
 
-For the plugin are called by their name in their profile.  To check for a plugin's profile name - for plugins built by external teams, please go to [https://github.com/ethereum/remix-plugins-directory/tree/master/plugins](https://github.com/ethereum/remix-plugins-directory/tree/master/plugins)
-
-## Further Customization with URL parameters
+### Deactiving a list of plugins
+```
+https://remix.ethereum.org/?#deactivate=udapp
+```
 
-The following URL will close everything except the main panel & the icon panel (so the side and terminal are minimized)
+Note: plugin are called by their name in their profile.  To check for a plugin's profile name - for plugins built by external teams, please go to [https://github.com/ethereum/remix-plugins-directory/tree/master/plugins](https://github.com/ethereum/remix-plugins-directory/tree/master/plugins)
 
-**https://remix.ethereum.org/?#embed=true**
+### Minimizing Remix panels
 
-To link with the side panel minimized use this URL:
+The following URL will **close everything except the main panel & the icon panel** (the side and terminal are minimized).
+```
+https://remix.ethereum.org/?#embed=true
+```
 
-**https://remix.ethereum.org/?#minimizesidepanel=true**
+To minimize just the side panel, use this URL:
+```
```

Comment:
it is also possible to minimize the terminal independently: `https://remix.ethereum.org/?#minimizeterminal=true`